.github/workflows/build-images.yml

```diff
@@ -0,0 +1,70 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
+      - latest
```

Comment:
Only for edge branch, not latest (which is done by `release` instead)

Reply:
Done